solidity/contracts/middleware/InterchainAccountRouter.sol

```diff
@@ -745,6 +754,47 @@ contract InterchainAccountRouter is Router {
             );
     }
 
+    function sendCommitment(
+        uint32 _destination,
+        bytes32 _router,
+        bytes32 _ism,
+        bytes memory _hookMetadata,
+        IPostDispatchHook _hook,
+        bytes32 _salt,
+        bytes32 _commitment
+    ) public payable returns (bytes32) {
```

Comment:
wdyt of `callRemoteShielded` or something?

Reply:
I also think we should have a version here which uses the default router, ism, hook, and hookMetadata

Reply:
like the simplest usage should be
```solidity
callRemoteShielded(uint32 _destination, bytes32 _commitment)
```